config/application.rb

```diff
@@ -13,12 +13,12 @@ class Application < Rails::Application
     config.time_zone = 'UTC'
     config.middleware.use Rack::Attack
     config.browserify_rails.force = true
-    config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
+    config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] --plugins [ transform-es3-member-expression-literals transform-es3-property-literals ]'
```

Comment:
`%w` will not work here. It assumes that each set of characters separated by a space is a string element in the array.
This:
```ruby
%w(-t [ babelify)
```
will result in this:
```ruby
["-t", "[", "babelify"]
```